binding/python/README.md

```diff
@@ -13,7 +13,7 @@ Falcon is an on-device speaker diarization engine. Falcon is:
 
 ## Compatibility
 
-- Python 3.9+
+- Python 3.10+
```

Comment:
We are testing on Python 3.9 in workflow and have 3.9 set in the binding setup.py, is this actually true?